src/App.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import './App.css';
+import { ToastContainer } from 'react-toastify';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import ProtectedRoute from './components/ProtectedRoute';
+import ProtectedRouteAdmin from './components/ProtectedRouteAdmin';
+import Home from './routes/Home';
+import Login from './routes/Login';
+import SignUp from './routes/SignUp';
+import AppointmentList from './routes/AppointmentList';
+import Doctors from './routes/Doctors';
+import DoctorDetail from './routes/DoctorDetail';
+import AddDoc from './routes/AddDoc';
+import DeleteDoc from './routes/DeleteDoc';
+import AppointmentUpdate from './routes/AppointmentUpdate';
+import CreateAppointment from './routes/CreateAppointment';
+import Layout from './components/Layout';
+import NotMatch from './routes/NotMatch';
+import UnAuthorize from './routes/UnAuthorize';
+import Logout from './routes/Logout';
+
+const queryClient = new QueryClient();
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 4):

![image](https://github.com/Atril33/DocConnect-Frontend/assets/57421823/66d5479c-8412-48a6-8b82-d486dba89ed8)


That way, your code will meet the latest standards. 💪